src/ios/FabricPlugin.m

```diff
@@ -246,6 +247,11 @@ - (void)sendCrash:(CDVInvokedUrlCommand*)command
     [[Crashlytics sharedInstance] crash];
 }
 
+- (void)recordError:(CDVInvokedUrlCommand*)command
+{
+    [[Crashlytics sharedInstance] recordError:error]
```

Comment:
This method requires an `NSError` object. Also, the error variable does not exist so this is a compilation error.